runtime/js/99_main.js

```diff
@@ -531,7 +531,14 @@ const finalDenoNs = {
   internal: internalSymbol,
   [internalSymbol]: internals,
   resources: core.resources,
```

Comment:
Should we also deprecate `Deno.resources()`?

Reply:
no, its still useful, especially since we still will have the test resource sanitizer

Reply:
What do you mean?

Reply:
Being able to see which resources are open is extremely useful, and it would be weird to remove it while we still have the resource sanitizer in the test runner.
Maybe the output of `Deno.resources()` should be changed (to just return an array), but lets leave it as-is for now

Reply:
I'm currently re-writing the resource sanitization stuff to work in Rust so I wonder it's better to deprecate this now so we can make it unstable in 2.0.

Reply:
Do you mean deprecate now to remove in v2?